src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -1,147 +1,175 @@
+        duration: event.target.elements.duration.value,
+        authors: [authorId],
+      })
+    );
+
+    navigate('/courses');
+  }
 
   return (
-    <Form
-      onSubmit={handleSubmit}
-      id='createCoursePortal'
-      style={{ display: 'none' }}
-    >
-      <Form.Group className='mb-3' controlId='formBasicTitle'>
-        <Form.Label>Title:</Form.Label>
-        <Form.Control
-          type='text'
-          name='title'
-          value={inputs.title || ''}
-          onChange={handleChange}
-        />
-        <Form.Text className='text-muted'>
-          Course title must be provided.
-        </Form.Text>
-      </Form.Group>
-      <Form.Group className='mb-3' controlId='formBasicDescription'>
-        <Form.Label>Description:</Form.Label>
-        <Form.Control
-          type='text'
-          name='description'
-          value={inputs.description || ''}
-          onChange={handleChange}
-        />
-        <Form.Text className='text-info'>
-          Course description must be provided.
-        </Form.Text>
-      </Form.Group>
-      <Form.Group className='mb-3' controlId='formBasicDuration'>
-        <Form.Label>Duration:</Form.Label>
-        <Form.Control
-          type='text'
-          name='duration'
-          value={inputs.duration || ''}
-          onChange={handleChange}
+    <form onSubmit={handleFormSubmit}>
+      <div className={styles.createCoursePortal}>
+        <div className={styles.createCourseForm}>
+          <div className={styles.titleInput}>
+            <Input
+              labelText={'Title'}
+              placeholderText={'Enter Title...'}
+              inputName={'title'}
+              inputType={'text'}
+              isRequired={true}
+            />
+          </div>
+          <Button buttonText={'Create course'} btnType={'submit'} />
+        </div>
+        <Input
+          className={styles.descriptionInput}
+          labelText={'Description'}
+          placeholderText={'Enter Description..'}
+          inputName={'description'}
+          inputType={'textarea'}
+          isRequired={true}
+          minLength={'2'}
         />
-        <Form.Text className='text-warning'>
-          Course duration must be provided.
-        </Form.Text>
-      </Form.Group>
-      <div className={styles.addAuthorPortal}>
-        <div className={styles.addAuthorPortalLeft}>
-          <Input
-            labelText='Author name'
-            placeholderText='Enter author name..'
-            name='author'
-            value={authorInput}
-            onChange={handleAuthorChange}
-          />
-          <Button buttonText='Create author' onClick={createAuthor} />
+        <div className={styles.labels}>
+          <h2>Add author</h2>
+          <h2>Authors</h2>
+        </div>
+        <div className={styles.createAuthorForm}>
+          <div className={styles.createAuthorFormHeader}>
+            <Input
+              labelText={'Author name'}
```

Comment:
Just an advice: it can be simplified for string props
```suggestion
              labelText="Author name"
```